src/main/scala/beam/utils/csv/writers/PlansCsvWriter.scala

```diff
@@ -74,6 +75,7 @@ object PlansCsvWriter extends ScenarioCsvWriter {
 
         val route = Option(leg.getRoute)
         PlanElement(
+          tripId = "leg.getAttributes.getAttribute(\"trip_id\").toString",
```

Comment:
where this string value will be evaluated/executed and replaced with the actual "trip_id"?